openstack/keystone/values.yaml

```diff
@@ -194,7 +194,7 @@ tempest:
   enabled: false
   image: "rally-openstack"
   #imageTag: "ccloud-1.5.0"
-  imageTag: "ccloud-zed"
+  imageTag: "ccloud-bobcat"
```

Comment:
does this tag exist? I suggest we set the value to "", and let decide the tag in per-region values.